cdk/lib/prism.ts

```diff
@@ -87,7 +87,7 @@ export class PrismStack extends GuStack {
         },
       },
       healthCheck: HealthCheck.elb({
-        grace: Duration.seconds(600),
+        grace: Duration.seconds(800),
```

Comment:
Could we use [`Duration.minutes`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_core.Duration.html#static-minutesamount) here instead?

Reply:
The docs, CFN, console all use seconds so I'm not sure it helps us that much to use a different unit here, especially with 800 which doesn't convert into a whole number of minutes (although admittedly 800 was a slightly arbitrary choice!)

Reply:
Ah! `Duration.minutes(8).toSeconds()` would work? Not massively more readable though esp if the duration isn't divisible by 60.

Reply:
You can do `Duration.minutes(13.3)` and `cdk` will convert it to 798 seconds, but I don't think this really makes things better as the console will still show 798 (and I'll still end up having to Google the conversion 😁)!